library/spdm_requester_lib/libspdm_req_challenge.c

```diff
@@ -29,6 +29,7 @@ typedef struct {
  * this function also perform the basic mutual authentication.
  *
  * @param  spdm_context           A pointer to the SPDM context.
+ * @param  reserved               Reserved for session_id.
```

Comment:
The internal functions don't need it yet. Just the public ones.

Reply:
The change for internal function is restored.